core-plugins/src/main/java/co/cask/hydrator/plugin/transform/ProjectionTransform.java

```diff
@@ -316,6 +336,8 @@ private Schema getOutputSchema(Schema inputSchema) {
       String inputFieldName = inputField.getName();
       if (fieldsToDrop.contains(inputFieldName)) {
         continue;
+      } else if (!fieldsToKeep.isEmpty() && !fieldsToKeep.contains(inputFieldName)) {
```

Comment:
so let's say there's a typo in the fields list, say the input schema has the fields "userid, score"  i want to keep score field, but have a typo "scorw".. so we have a field to keep which is not present in the input schema, should we throw an exception in this case?